tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -882,14 +882,24 @@ def _select_case(self, role, case):
         finder = re.compile(r'.*\b' + case + r'\b')
         finder_dotted = re.compile(r'.*\b' + case.replace(' ', r'\.') + r'\b')
         for elem in elems:
-            action_chains = ActionChains(self._browser)
-            action_chains.move_to_element(elem)
-            action_chains.perform()
-            logger.debug(elem.text)
-            if finder.match(elem.text) or finder_dotted.match(elem.text):
-                parent = elem.find_element_by_xpath('..')
-                checkbox = parent.find_element_by_class_name('tree-checkbox')
-                break
+            # elem.txt might be null when the required reference devices could
+            # not be met (either due to the quantity or the type) for specific test.
+            # perform() will throw exceptions if elem.text is null since Chrome
+            # and chromedriver 80. If elem is not shown in current case list
+            # window, move_to_element() will not work either.
+            if elem.text:
+                # execute a javascript to scroll the window to the elem
+                self._browser.execute_script('arguments[0].scrollIntoView();',
+                                             elem)
+                action_chains = ActionChains(self._browser)
+                action_chains.move_to_element(elem)
```

Comment:
Is this still necessary with line 892 added?

Reply:
no, 
ERROR:root:case[Router_5_1_11] error
Traceback (most recent call last):
  File "C:\GRL\Thread1.1\Python27\lib\unittest\case.py", line 329, in run
    testMethod()
  File "autothreadharness\harness_case.py", line 1177, in test
    self._select_case(self.role, self.case)
  File "autothreadharness\harness_case.py", line 899, in _select_case
    raise Exception('Failed to find the case')
Exception: Failed to find the case